pages/integrations/sso.md

```diff
@@ -73,6 +73,23 @@ You can configure the session duration to any timeout between 6 hours and 8,760
 
 <%= image "session_duration/configure_session_duration.png", width: 623, height: 315, alt: "Screenshot of the Buildkite SSO Session Duration Configuration" %>
 
+## SSO session IP address pinning
+
+> 📘 Enterprise feature
+> Pinning SSO sessions to IP addresses is only available on an [Enterprise](https://buildkite.com/pricing) plan.
+
+Prompt your users to re-authenticate when their IP address changes.
+
+With session IP address pinning, authorized sessions can only come from the IP address that created the session. If another IP address attempts to access Buildkite, the session will be immediately revoked. This helps to prevent session hijacking.
+
+To set up SSO session IP address pinning, use the [GraphQL API](/docs/apis/graphql/graphql-cookbook#pin-sso-sessions-to-ip-addresses) or complete the following steps in the Buildkite dashboard:
+
+* Select Provider
+* Under Session IP Address Pinning section
+  Click Update Session IP Address Pinning
+* Check Session IP Address Pinning checkbox
+* Click Save Session IP Address Pinning
```

Comment:
Since there's an order to these steps, let's make this an ordered list. I've also included the context about where things happen when screens change.

```suggestion
1. Navigate to the [organization's SSO settings](https://buildkite.com/organizations/~/sso).
1. In the _Configured SSO Providers_ section, select the provider.
1. In the _Session IP Address Pinning_ section, select _Update Session IP Address Pinning_.
1. In the modal that appears, select the _Session IP Address Pinning_ checkbox.
1. Select _Save Session IP Address Pinning_.
```